src/Native/Windows/build-native.cmd

```diff
@@ -111,6 +111,13 @@ echo See: https://github.com/dotnet/coreclr/blob/master/Documentation/project-do
 exit /b 1
 
 :GenVSSolution
+:: Generate Version file
+if not exist "%__IntermediatesDir%\clrcompression" md "%__IntermediatesDir%\clrcompression"
+msbuild /t:GenerateVersionHeader build.proj /p:NativeVersionHeaderFile="%__IntermediatesDir%\clrcompression\_version.h" /p:GenerateVersionHeader=true
+:: Copy NativeVersion.rc to the Intermediate dir
+if [%TOOLS_DIR%]==[] set TOOLS_DIR=%__binDir%\..\Tools\
+if not exist "%__IntermediatesDir%\clrcompression\NativeVersion.rc" copy "%TOOLS_DIR%NativeVersion.rc" "%__IntermediatesDir%\clrcompression\NativeVersion.rc"
```

Comment:
Instead of doing this dance, could we have the CMakeLists.txt for clrcompression just pull the rc file from the TOOLS_DIR?  That way if we add more native code in the future we don't have to copy the .rc file to each intermediate folder.